project-generator/src/test/java/io/syndesis/project/converter/DefaultProjectGeneratorTest.java

```diff
@@ -60,6 +60,7 @@
 
 @RunWith(Parameterized.class)
 public class DefaultProjectGeneratorTest {
+    private static final String CONNECTORS_VERSION = "0.5.6";
```

Comment:
Can we pass this as a system property to surefire rather than having to update each time we change the connector version?

Reply:
That was my first intention, but opted to make it as simple as possible, let me revisit this 👍